packages/core/src/modules/indy/repository/AnonCredsCredentialDefinitionRecord.ts

```diff
@@ -1,39 +1,47 @@
+import type { DidIndyNamespace } from '../../../utils/indyIdentifiers'
 import type { CredDef } from 'indy-sdk'
 
 import { BaseRecord } from '../../../storage/BaseRecord'
-import { didFromCredentialDefinitionId } from '../../../utils/did'
+import { getQualifiedIdentifier } from '../../../utils/indyIdentifiers'
+import { CredentialDefinitionTransformer } from '../../../utils/transformers'
+import { uuid } from '../../../utils/uuid'
 
 export interface AnonCredsCredentialDefinitionRecordProps {
   credentialDefinition: CredDef
+  didIndyNamespace: DidIndyNamespace
+  schemaSeqNo: number
 }
 
-export type DefaultAnonCredsCredentialDefinitionTags = {
-  credentialDefinitionId: string
```

Comment:
I think we want to keep the credentialDefinition as a tag?

Reply:
I'll have a look

Reply:
hmm not sure why this si showing here. ALready removed this in my last commit (locally)